apps/builder/app/shared/canvas-api.ts

```diff
@@ -0,0 +1,73 @@
+import { setInert, resetInert } from "../canvas/shared/inert";
+
+const apiWindowNamespace = "__webstudio__$__canvasApi";
+
+const _canvasApi = {
```

Comment:
if we have a nicely prefixed namespace, why do you need a _ before the name here?